pkg/sql/sql_test.go

```diff
@@ -262,6 +262,36 @@ var _ = Describe("Sql", func() {
 		})
 	})
 
+	Describe("#ListKubernetesResourceNamesByAccountNameAndKindAndNamespace", func() {
+		var names []string
+
+		JustBeforeEach(func() {
+			names, err = c.ListKubernetesResourceNamesByAccountNameAndKindAndNamespace("test-account-name",
+				"test-kind", "test-namespace")
+		})
+
+		When("it succeeds", func() {
```

Comment:
instead of it succeeds can you say when("account_name, kind, and namespace are provided")

Reply:
Looking this over, it doesn't really align with the rest of the package.

If we're unhappy with the success test case notation of "it succeeds" then we should probably find something generic and replace them project-wide.

The problem I see with your example is it doesn't necessarily provide for the success case. Just because accountName, kind, and namespace are provided does not guarantee a non-error response from the database.